CONTRIBUTING.md

```diff
@@ -0,0 +1,324 @@
+| _model_name_                            | Folder containing the modeling, preprocess, config, train, and eval scripts.                                                                                                                                                                    |
+| _model_name/config_                     | Folder containing the JSON configuration files used for the train and eval scripts.                                                                                                                                                             |
+| _model_name/config.py_                  | Script containing the model config class which inherits from HuggingFace's [`PretrainedConfig`](https://huggingface.co/transformers/main_classes/configuration.html#transformers.PretrainedConfig) or its family of derived classes.            |
+| _model_name/eval.py_                    | Script containing code to evaluate the model performance.                                                                                                                                                                                       |
+| _model_name/modeling.py_                | Script containing the model class which inherits from HuggingFace's [`PretrainedModel`](https://huggingface.co/transformers/main_classes/model.html#transformers.PreTrainedModel) class or its family of derived classes.                       |
+| _model_name/preprcoess.py_              | Script containing code to preprocess the input text into the model's required input tensors.                                                                                                                                                    |
+| _model_name/tokenization.py_ (optional) | Script containing the model tokenizer class which inherits from HuggingFace's [`PretrainedTokenizer`](https://huggingface.co/transformers/main_classes/tokenizer.html#transformers.PreTrainedTokenizer) class or its family of derived classes. |
+| _model_name/train.py_                   | Script containing code to train the model. It is recommended to utilize the [`Trainer`](https://huggingface.co/transformers/main_classes/trainer.html#transformers.Trainer) class from HuggingFace.                                             |
+| _model_name/README.md_                  | README markdown file containing model information such as model source, architecture, evaluation datasets and metrics, model size, and training information.                                                                                    |
+| _model_name/requirements.txt_           | Text file containing package dependencies for model serving. To manage the full list of dependencies in `sgnlp`, contributors are strongly recommended to limit their code to use the packages listed in `setup.py`.                            |
```

Comment:
Inconsistent in starting with sgnlp as root and describing sgnlp/sgnlp while the rest don't have the full path.
Actually with this, the above section on project structure might not be necessary. I don't think describing the root and main folder is particularly important. Can go straight to the models folder. Say it can be found in sgnlp/models.

Reply:
Typo in preprocess.py

Reply:
The requirements.txt is outdated actually. We don't want it. If there's extra dependencies involved that's out of the list indicated in setup.py, should be indicated in __init__.py